<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UltrasoundSupportControls</class>
 <widget class="QWidget" name="UltrasoundSupportControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Device Management</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="m_Headline">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Connected Devices:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkUSDeviceManagerWidget" name="m_DeviceManagerWidget" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_AddDevice">
         <property name="text">
          <string>New Device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkUSNewVideoDeviceWidget" name="m_NewVideoDeviceWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>US Imaging</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Active Ultrasound Devices:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkServiceListWidget" name="m_ActiveVideoDevices" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_BtnView">
         <property name="text">
          <string>Start Viewing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_FramerateLabel">
         <property name="text">
          <string>Current Framerate: 0 FPS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Framerate Limit:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_FrameRate">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>(Restart viewing to make changes effective)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkUSDeviceManagerWidget</class>
   <extends>QWidget</extends>
   <header>QmitkUSDeviceManagerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkUSNewVideoDeviceWidget</class>
   <extends>QWidget</extends>
   <header>QmitkUSNewVideoDeviceWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkServiceListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkServiceListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
